### DISCUSSION TITLE
Skip a query

### DISCUSSION THREAD

**Discussion by BETZCorentin** [the-guild-org/apollo-angular#1639] [Q&A]
I saw that there is an option « skip » for queries in react but not in angular. I wanted to know if there is another way to skip a query in apollo ?

**Comment by Trottero** [comment]
Could this be looked at?

I need advanced control over when a query is being invoked, but it seems that no matter what I do, as soon as I instantiate a watch query and subscribe to its `valuechanges` it invokes the query.
